build/Community.Toolkit.Common.targets

```diff
@@ -12,9 +12,9 @@
   </PropertyGroup>
 
   <ItemGroup Condition="$(IsPackable)">
-    <None Include="$(BuildToolsDirectory)nuget.png" Pack="true" PackagePath="\Icon.png" Visible="False" />
-    <None Include="$(RepositoryDirectory)License.md" Pack="true" PackagePath="\" Visible="False" />
-    <None Include="$(RepositoryDirectory)ThirdPartyNotices.txt" Pack="true" PackagePath="\" Visible="False" />
+    <None Pack="true" PackagePath="\" Visible="False" Include="$(BuildToolsDirectory)Icon.png" />
```

Comment:
I want, with all of my being, to start with `Include` but doesn't **this look nice** and it's not like we're going to update this anytime soon!

Reply:
No it doesn't, please just leave `Include` first 😆
It's also alphabetically in order, I don't understand why you would want to put that last at all.

Reply:
I actually forgot why I changed this, since it was a long time ago. But thanks to [this comment](#discussion_r888842771), I found that I missed a commit during rebase from the split from the **WindowsCommunityToolkit** repo. It is now included here.